src/parse.c

```diff
@@ -1295,7 +1295,7 @@ handle_routes(yaml_document_t* doc, yaml_node_t* node, const void* _, GError** e
         cur_route->type = g_strdup("unicast");
         cur_route->scope = g_strdup("global");
         cur_route->family = G_MAXUINT; /* 0 is a valid family ID */
-        cur_route->metric = G_MAXUINT; /* 0 is a valid metric */
+        cur_route->metric = METRIC_UNSPEC; /* 0 is a valid metric */
```

Comment:
I didn't see METRIC_UNSPEC get a define, is this a glib magic suffix?

Reply:
No; we added it some time ago. I think here it was just missed; I updated that line for clarity / consistency.

Reply:
Indeed, I see it now in the source.  Thanks